### DISCUSSION TITLE
Remote administration of Tier2 with d365fo.tools?

### DISCUSSION THREAD

**Discussion by zmottie** [d365collaborative/d365fo.tools#600] [Q&A]
Is it possible to use these tools (specifically  the user related tools such as import-d365aaduser) against Tier2/Prod environments? If so can you please point me to a document/guide that explains how to do it? 

I've used them with great success against CHE but when I attempt to run them from my local machine against Tier2 I simply get 'Cannot bind argument to parameter' errors. My assumption is I need to 'Enable access' from within LCS but I couldn't find anything in any of the examples/documents to support this so I thought I'd ask here before doing anything hasty.

thanks!

**Comment by Splaxi** [comment]
The landscape around Tier2 has changed a lot since the beginning of the tools. 

If you have a valid Just In Time access from LCS and whitelisted your ip address - it would technically work against a Tier2.

PROD will not work. 

You could go to the sibling module, d365fo.integrations. This will help you access the OData endpoint - where you can import users and configure assignments of security roles. We don't have a step-by-step guide, but I know it works, as several of my customers use it. PROD would work with this OData approach.

**Comment by Splaxi** [comment]
Some inspiration

https://d365fostuff.wordpress.com/2021/04/06/d365fo-integration-enable-users-after-database-refresh/amp/
